views/dashboard.handlebars

```diff
@@ -1,4 +1,26 @@
 {{> nav}}
 
+<div class="container">
+    <div class="hero ">
+        <div class="hero-body" >
+            <img src="/images/gamers_game.png" alt="banner with gaming consol controllers">
+        </div>
+    </div>
+    <div class="hero is-primary has-text-black is-medium">
+        <div class="hero-body has-text-centered">
+            <h1 class="title"> Hello {{userDBname}} !</h1>
```

Comment:
What's wrong with just username?

Reply:
isn't bad if we are using same const names all over again?!
This way it it is kind "info from DB"